man/wvunpack.1

```diff
@@ -172,18 +174,18 @@ about WavPack file to
 .Pa stdout
 .It Fl -skip=[-][ Ns Ar sample Ns | Ns Ar hh : Ns Ar mm : Ns Ar ss.ss ]
 start decoding at specified sample or time index, specifying a
-.Fl
+.Sy -
```

Comment:
I's say `.Sq -` as it's a literal minus char.

Reply:
Yes, `.Sq` is a better fix. The `.Fl` was okay for '-' but not for the '+' where it displayed '-+'

Reply:
The Fl is actualy wrong for even the `-`, as it's not a flag, but a literal minus; so your edit is right.